code/game/machinery/vending/vendor_types/squad_prep/squad_engineer.dm

```diff
@@ -9,8 +9,10 @@ GLOBAL_LIST_INIT(cm_vending_gear_engi, list(
 		list("JIMA Planted Flag", 0, /obj/item/defenses/handheld/planted_flag, MARINE_CAN_BUY_ATTACHMENT, VENDOR_ITEM_MANDATORY),
 		list("UA 42-F Sentry Flamer", 0, /obj/item/defenses/handheld/sentry/flamer, MARINE_CAN_BUY_ATTACHMENT, VENDOR_ITEM_MANDATORY),
 		list("UA 571-C Sentry Gun", 0, /obj/item/defenses/handheld/sentry, MARINE_CAN_BUY_ATTACHMENT, VENDOR_ITEM_MANDATORY),
+		list("M6H-BRUTE",0, /obj/item/storage/belt/gun/brutepack/full, MARINE_CAN_BUY_ATTACHMENT, VENDOR_ITEM_MANDATORY),
```

Comment:
Should give you a hint of what it does immediately, see how above is "sentry gun", for example; "M6H-BRUTE Breaching Launcher" or similar.